#! /bin/sh /usr/share/dpatch/dpatch-run
## 00_use_local_shebang.dpatch by  <cb@xmcorsairs.wu-wien.ac.at>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: remove /usr/local from #!

@DPATCH@

diff -Naur CGIwithR.orig/inst/examples/dangerous.R CGIwithR/inst/examples/dangerous.R
--- CGIwithR.orig/inst/examples/dangerous.R	2005-11-23 16:07:08.000000000 +0000
+++ CGIwithR/inst/examples/dangerous.R	2008-09-13 15:11:14.000000000 +0000
@@ -1,4 +1,4 @@
-#! /usr/local/bin/R  
+#! /usr/bin/R  
 
 ###  An example CGI script in R
 ###
diff -Naur CGIwithR.orig/inst/examples/trivial.R CGIwithR/inst/examples/trivial.R
--- CGIwithR.orig/inst/examples/trivial.R	2005-11-23 16:07:08.000000000 +0000
+++ CGIwithR/inst/examples/trivial.R	2008-09-13 15:11:24.000000000 +0000
@@ -1,4 +1,4 @@
-#! /usr/local/bin/R
+#! /usr/bin/R
 
 tag(HTML)
     tag(HEAD)
